### DISCUSSION TITLE
Spellbook swap counter

### DISCUSSION THREAD

**Discussion by roso1** [runelite/runelite#15455] [Ideas]
Doing the command !sbs will tell you how many spell book swaps you have left for the day.

**Comment by MESLewis** [comment]
Since this isn't something that needs to be shared with others it should be "::sbs" and not "!sbs"

**Comment by Enriath** [comment]
If this is for the magic skillcape, this should just be an item charge displayed on the magic skillcape itself

**Reply by roso1** [reply]
Well I had the idea when I was going to nex getting kc, I was on ancients and wanted to switch to arceaus, I didn't know if I had any swaps left since I didn't have the cape on me or anything
